Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1190,6 +1190,15 @@ void ScheduleReads(u64 dvd_offset, u32 dvd_length, bool decrypt, u32 output_addr
   // If we fall within its bounds, we get DMA-speed reads.
   u64 buffer_start, buffer_end;
 
+  // The variable offset uses the same addressing as games do.
```

Comment:
See below for more context, but I'd suggest that we omit the changes to this method. In fact, I'd move this code to the start of `ExecuteReadCommand` instead.